packages/modeling/src/operations/extrusions/extrudeFromSlices.js

```diff
@@ -54,6 +55,9 @@ const extrudeFromSlices = (options, base) => {
 
   if (numberOfSlices < 2) throw new Error('numberOfSlices must be 2 or more')
 
+  // Repair gaps in the base slice
```

Comment:
Why the repair? Were there some cases where the other extrusion functions were creating invalid slices?

Reply:
The repair is necessary to make common examples work. Many of the existing jscad functions, like the BSP trees, will often return non-closed geometries. The birthday balloons, for example, produces a non-closed slice. I felt like it was important to not break existing designs, and so repair seemed necessary.